cypress/support/parent-commands.ts

```diff
@@ -79,13 +79,15 @@ declare namespace Cypress {
      * - on click of header, the changelog content is visible.
      */
     checkChangelogRevealer(): void;
+
+    abortPrefetchRequests(target: string): void;
   }
 }
 
 Cypress.Commands.add('getDocsPageContentArea', () => cy.get('#paste-docs-content-area'));
 
 Cypress.Commands.add('pageHeaderShouldBeVisible', (headerText) => {
-  cy.contains('h1', headerText).shouldBeVisible();
+  cy.contains('h1', headerText, {matchCase: false}).should('be.visible');
```

Comment:
I'm starting to wonder why these test assertions are about visibility. We don't do anything about it being visible, we are trying to assert they exist.